core/src/main/scala/com/indoorvivants/vcpkg/package.scala

```diff
@@ -0,0 +1,13 @@
+package com.indoorvivants
+
+import java.io.File
+
+package object vcpkg {
+
+  private val FILE_SEPARATOR = System.getProperty("file.separator");
+
+  private[vcpkg] implicit class FileOps(private val file: File) extends AnyVal {
+    def /(s: String): File = new File(file.getAbsolutePath() + FILE_SEPARATOR + s)
```

Comment:
not sure if that's the best way 🤷

Reply:
I think you can do `Paths.get(file.toPath, s).toFile`